saleor/graphql/product/sorters.py

```diff
@@ -123,7 +123,7 @@ class Meta:
 
 class ProductOrderField(graphene.Enum):
     NAME = ["name", "slug"]
-    RANK = ["name", "slug"]
+    RANK = ["search_rank"]
```

Comment:
We should add a unique field as the last order field. Without it, pagination couldn't work. 
```suggestion
    RANK = ["search_rank", "slug"]
```